utils/test_results.py

```diff
@@ -359,16 +371,21 @@ def generate_test_results(
     if parameter is not None:
         match parameter:
             case GENERATE_TEST_RESULT_PARAM.FLAKY:
-                flakes = Flake.objects.filter(
-                    Q(repository_id=repoid)
-                    & (Q(end_date__date__isnull=True) | Q(end_date__date__gt=since))
-                )
+                totals = get_relevant_totals(repoid, branch, since)
```

Comment:
Reading it this way now, we could also bump totals up to line 372 and use it for each case now right?

Vs. having it be the "starting line" in each of our case statements below